### ISSUE TITLE
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.6:sign (sign-artifacts) on project parallel-test-runner-parent: Exit code: 2

### ISSUE THREAD

**Issue by fermenreq** [diva-e/parallel-test-runner#1]
**Hi:**

**I clone the project and I followed the instructions on a README file to install it, and I got the next ERROR:** 

```
$ mvn install
[INFO] Scanning for projects...
[INFO] Inspecting build with total of 3 modules...
[INFO] Installing Nexus Staging features:
[INFO]   ... total of 3 executions of maven-deploy-plugin replaced with nexus-staging-maven-plugin
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] Parallel Test Runner Parent
[INFO] Parallel Test Runner
[INFO] Parallel Test Runner Demo
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building Parallel Test Runner Parent 1.1-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-source-plugin:3.0.1:jar-no-fork (attach-sources) @ parallel-test-runner-parent ---
[INFO]
[INFO] --- maven-javadoc-plugin:2.10.4:jar (attach-javadocs) @ parallel-test-runner-parent ---
[INFO] Not executing Javadoc as the project is not a Java classpath-capable package
[INFO]
[INFO] --- maven-gpg-plugin:1.6:sign (sign-artifacts) @ parallel-test-runner-parent ---
gpg: skipped "B5616A11": secret key not available
gpg: signing failed: secret key not available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Parallel Test Runner Parent ........................ FAILURE [  1.793 s]
[INFO] Parallel Test Runner ............................... SKIPPED
[INFO] Parallel Test Runner Demo .......................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.983 s
[INFO] Finished at: 2017-04-25T12:06:36+02:00
[INFO] Final Memory: 17M/227M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.6:sign (sign-artifacts) on project parallel-test-runner-parent: Exit code: 2 -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException


```

**Comment by ckumpe** [comment]
Hi fermenreq,

I can reproduce the error.  Thanks for your hint. I'll try to fix in the next days.

**Comment by ckumpe** [comment]
it's fixed in the master branch in e198e6e26188bca9949a3880d153fb8ceb1ff988
